bpf/include/bpf_helper_defs.h

```diff
@@ -0,0 +1,212 @@
+// Autogenerated from libbpf sources
+static void *(* const map_lookup_elem)(void *map, const void *key) __maybe_unused = (void *) 1;
+static long (* const map_update_elem)(void *map, const void *key, const void *value, __u64 flags) __maybe_unused = (void *) 2;
+static long (* const map_delete_elem)(void *map, const void *key) __maybe_unused = (void *) 3;
```

Comment:
Both of these functions have changed their return type from `int` to `long`.
Since libbpf is the source of truth, this should be ok (also: https://docs.ebpf.io/linux/helper-function/bpf_map_update_elem/)